### ISSUE TITLE
Getting "bootstrap-accessibility.js:637 Uncaught TypeError: Cannot read property 'Constructor' of undefined" some conflict with carousel component.  Custom build w/o carousel?

### ISSUE THREAD

**Issue by dondmcg** [paypal/bootstrap-accessibility-plugin#126]
Can I get/create a custom build of this plugin without the carousel component?  I need this to test if that component is the cause of the conflict.

**Comment by mpnkhan** [comment]
Just remove that from https://github.com/paypal/bootstrap-accessibility-plugin/blob/master/Gruntfile.js#L34 . and build
